Source/Converters/fsIEnumerableConverter.cs

```diff
@@ -47,7 +47,13 @@ public class fsIEnumerableConverter : fsConverter {
         }
 
         private bool IsStack(Type type) {
-            return type.IsGenericType && type.GetGenericTypeDefinition() == typeof(Stack<>);
```

Comment:
Thanks. There is a magic `Resolve()`  method which takes care of this, so can we just change this to

```
return type.Resolve().IsGenericType && type.GetGenericTypeDefinition() == typeof(Stack<>);
```